code/perpetualpayments/Makefile

```diff
@@ -0,0 +1,16 @@
+
+REGISTRY=docker.com
+NAME=lnbook
+CONTAINERS=bitcoind c-lightning
+
+all: build-all push-all
```

Comment:
Had a try today, would'nt it be better to default make to just create the containers and require an argument if it should be pushed to the registry. Don't know whether this will be part of the book resources though.

Reply:
I’m gonna delete the extra stuff I added and make another PR. The bump is necessary.